scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -279,6 +287,15 @@ def check(method, directed):
             check(method, directed)
 
 
+def test_negative_weights():
+    def check(method):
+        SP = shortest_path(directed_negative_weighted_G, method, directed=True)
+        assert_array_almost_equal(SP, directed_negative_weighted_SP)
+
+    for method in ['FW', 'J', 'BF']:
+        check(method)
```

Comment:
This test can be parameterised instead of the for loop
https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
Thank you.
Done.